app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command_split = command.split()
+
+    if len(command_split) == 3:
+        file_command = command_split[0]
+        original = command_split[1]
+        copy = command_split[2]
```

Comment:
You can use unpacking here
```suggestion
        file_command, original, copy = command_split
```